src/views/Joseki/Joseki.tsx

```diff
@@ -43,7 +43,7 @@ import { JosekiTagSelector, JosekiTag } from "JosekiTagSelector";
 import { Throbber } from "Throbber";
 import { IdType } from "src/lib/types";
 
-const server_url = data.get("oje-url", "/oje/");
+const server_url = "/oje/";
```

Comment:
Why are we doing this?

The reason for being able to change the url is in case you want to bring up a different joseki server... it certainly turned out  handy when migrating....

Reply:
Ah I didn't see a corresponding `data.set` call so I figured it was just obsolete. 

How do you use this generally? Mainly I'm wondering what is the benefit to checking localstorage rather than just modifying this line of code next time you need to talk to a different server..

Reply:
At any rate, added that back in!